tests/flow/test_enhanced_coverage.py

```
@@ -0,0 +1,293 @@
+        assert result.result_set[0][0] == 2
+        
+        # Clean up copy
+        copy_graph.delete()
+    
+    def test_error_handling_scenarios(self):
+        """Test various error handling scenarios"""
+        
+        # Test query with syntax error
+        try:
+            self.graph.query("INVALID SYNTAX QUERY")
+            assert False, "Should have raised syntax error"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)